modules/globebrowsing/src/renderableglobe.cpp

```diff
@@ -806,6 +807,10 @@ void RenderableGlobe::render(const RenderData& data, RendererTasks& rendererTask
     _lastChangedLayer = nullptr;
 }
 
+void RenderableGlobe::renderSecondary(const RenderData& data, RendererTasks&) {
+    _globeLabelsComponent.draw(data);
```

Comment:
The old `draw` was in a `try`/`catch` block;  are we potentially letting an exception escape here that might lead to a crash?

Reply:
Good catch ;)  I thought the try-catch was only related to the layer rendering, but the font rendering can also throw a texture unit error. 

Looking at the code in the existing catch block, I don't think we would have to do anything of this?  

![image](https://user-images.githubusercontent.com/8808894/221511048-68763bad-951d-4c7d-b247-2cb3dea218c8.png)

But instead maybe just print the message? 
![image](https://user-images.githubusercontent.com/8808894/221511087-30249080-6e46-4ae0-b4be-ed375503a506.png)